chat-plugins/faq.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const ROOMFAQ_FILE = path.resolve(__dirname, '../config/chat-plugins/faqs.json');
+
+let roomFaqs = {};
+try {
+	roomFaqs = require(ROOMFAQ_FILE);
+} catch (e) {} // file doesn't exist or contains invalid JSON
+if (!roomFaqs || typeof roomFaqs !== 'object') roomFaqs = {};
+
+function saveRoomFaqs() {
+	fs.writeFileSync(ROOMFAQ_FILE, JSON.stringify(roomFaqs));
+}
+
+exports.commands = {
+	addfaq: function (target, room, user) {
+		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (!this.can('declare', null, room)) return false;
+		if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
+		let [topic, ...text] = target.split(',');
+
+		if (!(topic && text.length)) return this.parse('/help roomfaq');
+		topic = toId(topic);
```

Comment:
Why is the ``toId(topic)`` done after the check for whether or not there is a topic?   
This would allow people to do ``/addfaq ( ͡° ͜ʖ ͡° ), dropped my bag of lenny faces`` and have it save as ``roomFaqs[room.id][""]``, which is not callable using /roomfaq and not deleteable with /removefaq